### PR TITLE
Yaeti

### PR DESCRIPTION
**Pull request by saraswati-patra**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [x] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by seth-priya** [comment]
@kumarvikas2020 please review

File: y/yaeti/yaeti_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+if ! npm install && npm audit fix && npm audit fix --force; then
+	echo "------------------$PACKAGE_NAME:install_fails-------------------------------------"
+	echo "$PACKAGE_URL $PACKAGE_NAME"
+	echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Install_Fails"
+	exit 1
+else
+	echo "------------------$PACKAGE_NAME:install_success &_test_not_available-------------------------"
+	echo "$PACKAGE_URL $PACKAGE_NAME"
+	echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Install_Fails"
```
Comment (kumarvikas2020) [inline y/yaeti/yaeti_8.5.sh:65]: @saraswati-patra Please check the error message. This is a success case.

Comment (saraswati-patra) [reply y/yaeti/yaeti_8.5.sh]: correction done,thanks.

Comment (kumarvikas2020) [reply y/yaeti/yaeti_8.5.sh]: @saraswati-patra There are too many commits. Can you squash the commits in a single one.

Comment (saraswati-patra) [reply y/yaeti/yaeti_8.5.sh]: @vikas ,before i had faced some extra file added issue ,in the added file deleted time not able delete all file at time so too many commits there. Now resolved that issue.so in this case i am going to delete branch ,close PR and will raise new PR for same package.

**Pull request closed without merging**